index.html

```diff
@@ -0,0 +1,56 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css" />
+    <title>Portfolio Project</title>
+</head>
+<body>
+
+    <div class="template-1">
+
+        <!-- Toolbar Nav -->
+    <nav class="toolbar-container">
+        <!-- Toolbar Name -->
+        <div class="toolbar-name-container">
+            <p class="toolbar-name">Wahab</p>
+        </div>
+        <!-- Toolbar Menu -->
+        <div class="toolbar-menu-container">
+            <img src="assets/Menu.svg" alt="Menu icon">
+        </div>
+    </nav>
+
+    <!-- Headline -->
+    <section class="headline-container">
+        <!-- Headline Text1 -->
+        <div class="headline-text1-container">
+            <p class="headline-text1">I’m Wahab.
+                Glad to see you!</p>
+        </div>
+
+        <!-- Headline Text2 -->
+        <div class="headline-text2-container">
+            <p class="headline-text2">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+        </div>
+
+        <!-- Headline Social media -->
+        <div class="headline-socialmedia-container">
+            <p class="headline-socialmedia-text">LET’S CONNECT</p>
+            <div class="headline-socialmedia-icons-container">
+                <img src="assets/twitter.svg" alt="Twitter icon">
+                <img src="assets/linkedin.svg" alt="Linkedin icon">
+                <img src="assets/medium.svg" alt="Medium icon">
+                <img src="assets/github.svg" alt="Github icon">
+                <img src="assets/victory.svg" alt="Hand icon">
```

Comment:
Good job on implementing this part 👌, only wrap these images in anchor tags and give them valid href to your social media accounts

Reply:
that's A good Suggestive Review.